frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTableConfig.tsx

```diff
@@ -99,178 +101,147 @@ const generateTableHeaders = (options: {
-          disableSortBy: true,
-          accessor: "name",
-          Cell: (cellProps: ICellProps): JSX.Element => (
+  const tableColumns: IDataColumn[] = [
+    {
+      title: "Name",
+      Header: "Name",
+      disableSortBy: true,
+      accessor: "name",
+      Cell: (cellProps: ICellProps): JSX.Element => (
+        <LinkCell
+          classes="w250-sm"
+          value={cellProps.cell.value}
+          path={PATHS.EDIT_POLICY(cellProps.row.original)}
+        />
+      ),
+    },
+    {
+      title: "Yes",
+      Header: () => <PassingColumnHeader isPassing />,
+      disableSortBy: true,
+      accessor: "passing_host_count",
+      Cell: (cellProps: ICellProps): JSX.Element => {
+        if (cellProps.row.original.has_run) {
+          return (
             <LinkCell
-              value={cellProps.cell.value}
-              path={PATHS.EDIT_POLICY(cellProps.row.original)}
-              classes="" // Override default
+              value={`${cellProps.cell.value} host${
+                cellProps.cell.value.toString() === "1" ? "" : "s"
+              }`}
+              path={
+                PATHS.MANAGE_HOSTS +
+                TAGGED_TEMPLATES.hostsByPolicyRoute(
+                  cellProps.row.original.id,
+                  PolicyResponse.PASSING,
+                  selectedTeamId
+                )
+              }
             />
-          ),
-        },
-      ];
-    default: {
-      const tableHeaders: IDataColumn[] = [
-        {
-          title: "Name",
-          Header: "Name",
-          disableSortBy: true,
-          accessor: "name",
-          Cell: (cellProps: ICellProps): JSX.Element => (
+          );
+        }
+        return (
+          <>
+            <span
+              className="text-cell text-muted has-not-run tooltip"
+              data-tip
+              data-for={`passing_${cellProps.row.original.id.toString()}`}
+            >
+              ---
+            </span>
+            <ReactTooltip
+              place="bottom"
+              effect="solid"
+              backgroundColor="#3e4771"
+              id={`passing_${cellProps.row.original.id.toString()}`}
+              data-html
+            >
+              {getTooltip(cellProps.row.original.osquery_policy_ms)}
+            </ReactTooltip>
+          </>
+        );
+      },
+    },
+    {
+      title: "No",
+      Header: () => <PassingColumnHeader isPassing={false} />,
+      disableSortBy: true,
+      accessor: "failing_host_count",
+      Cell: (cellProps: ICellProps): JSX.Element => {
+        if (cellProps.row.original.has_run) {
+          return (
             <LinkCell
-              classes="w250-sm"
-              value={cellProps.cell.value}
-              path={PATHS.EDIT_POLICY(cellProps.row.original)}
+              value={`${cellProps.cell.value} host${
+                cellProps.cell.value.toString() === "1" ? "" : "s"
+              }`}
+              path={
+                PATHS.MANAGE_HOSTS +
```

Comment:
Do we want to continue this TAGGED_TEMPLATES pattern or do we want to refactor this page to use `buildQueryStringFromParams`?

Reply:
done